bgpd/bgp_vty.c

```diff
@@ -15563,6 +15622,10 @@ int bgp_config_write(struct vty *vty)
 		if (bgp->autoshutdown)
 			vty_out(vty, " bgp default shutdown\n");
 
+		/* BGP instance administrative shutdown */
+		if (CHECK_FLAG(bgp->flags, BGP_FLAG_SHUTDOWN))
+			vty_out(vty, " bgp shutdown\n");
```

Comment:
I think we should save the shutdown message too, or it doesn't make sense?

I found this out while trying the `ALIAS` diff.

Reply:
does not really make sense to me, because if the daemon is restarted with a shutdown in the config, the peers are not brought up at all until the shutdown is lifted. The shutdown message is only important for already connected peers. In all honesty though, this command should probably not get written to the config by a user at all. I just put in to pass the unit tests, because config persistence is the first thing to get tested.

Reply:
@rzalamena: would you concern this as resolved?

Reply:
Yes, thank you for the explanation.